.travis.yml

```diff
@@ -21,7 +21,7 @@ before_deploy:
 - ./.travis/install_install4j
 - ENGINE_VERSION=$(grep "engine_version" game_engine.properties | sed 's/.*= *//g')
 - TAGGED_VERSION="$ENGINE_VERSION.$TRAVIS_BUILD_NUMBER"
-- ./.travis/build_installers $TAGGED_VERSION
+- TAGGED_VERSION=$TAGGED_VERSION ./gradlew release
```

Comment:
Sure...  Our Gradle build script has a data dependency on the `TAGGED_VERSION` environment variable (`build.gradle:14`).  Because `TAGGED_VERSION` is not exported on the previous line, it must explicitly be defined to appear in the environment of the Gradle process.

I think we used to export it when everything was in `.travis.yml`.  But, IIRC, when you broke the different concerns of `.travis.yml` into separate shell scripts (f687bc9), you made it a point to explicitly show the data dependencies (a Good Thing, IMO).  So I did it this way rather than prefixing the previous line with an `export`.

(Going through the history of `build_installers`, I see that I had to fix a build break in 4785d02 by exporting `TAGGED_VERSION` within the context of that script only.  I think I said in the PR at the time that the `TAGGED_VERSION=$TAGGED_VERSION` prefix looked messy, and I went with exporting `TAGGED_VERSION` instead.  As I said above, I didn't think that was an option here so we could keep the data dependencies explicit.)

Personally, I would prefer if we passed this value as a Gradle project variable (e.g. `./gradlew release -Pversion=$TAGGED_VERSION`), but I didn't want to do too much in this PR.